templates/password.json

```diff
@@ -4,13 +4,27 @@
     "main": {
       "type": "newsletter",
       "settings": {
-        "heading": "Opening soon",
-        "subheading": "<p>Be the first to know when we launch.</p>",
         "full_width": false
-      }
+      },
+      "blocks": {
+        "heading": {
+          "type": "heading",
+          "settings": {
+            "heading": "Opening soon"
+          }
+        },
+        "subheading": {
+          "type": "paragraph",
+          "settings": {
+            "paragraph": "<p>Be the first to know when we launch.</p>"
+          }
+        },
+        "email": {
```

Comment:
Same as my other comment. I have a personal preference for `email_form`—it describes the block.